packages/design-system/src/components/toast.tsx

```diff
@@ -383,9 +383,12 @@ export const Toaster = () => {
 type Options = Pick<ToastOptions, "duration" | "id" | "icon">;
 
 export const toast = {
-  info: (value: string, options?: Options) => hotToast(value, options),
-  error: (value: string, options?: Options) => hotToast.error(value, options),
-  warn: (value: string, options?: Options) => hotToast.custom(value, options),
-  success: (value: string, options?: Options) =>
+  info: (value: JSX.Element | string, options?: Options) =>
```

Comment:
why JSX.Element and not ReactNode?

Reply:
Because react-hot-toast does not accept ReactNode